### PR TITLE
Disable old UI

### LINKED ISSUE #8: Disable the old UI entierly
Remove the option to use the old UI completely. Supporting new functionalities for both will be a problem.

### PR DESCRIPTION
**Pull request by marq**
fixes #8

### CODE REVIEW

File: src/TQVaultAE.GUI/Properties/Resources.fr.resx [hunk 1]
```diff
@@ -991,13 +991,6 @@ Use &amp; after the name or type to filter by item rarity.
 For example, to filter only rare items append &amp;rare to your search string.
 The terms can also be combined @sword&amp;rare would only search for rare swords.</value>
   </data>
-  <data name="SettingsEnableNewUI" xml:space="preserve">
```
Comment (marq) [inline src/TQVaultAE.GUI/Properties/Resources.fr.resx]: I think it's fine. This line and first line below removed ones are the same. Diff is just displayed that way.

**Comment by EtienneLamoureux** [comment]
Think you can fix the conflict? I'll merge it in after.

**Comment by marq** [comment]
Conflict fixed.

**Comment by EtienneLamoureux** [comment]
Merged in as part of https://github.com/EtienneLamoureux/TQVaultAE/pull/27

**Pull request closed without merging**